tests/db/utils_test.py

```diff
@@ -723,8 +723,6 @@ def test_prefer_plaintext(self):
         self.assertEqual(actual, expected)
 
     # Mock the handler to cat, so that no transformations of the html are made
-    # making the result non-deterministic
```

Comment:
depending on the handler it may or may not make transformations to the HTML, which would result in the test failing. I think that it's fine to remove the comment though.

Reply:
oh i see. you mean the prefer-plaintext option. I suppose that it is safe to remove the mock here because we'd run test mostly in a sandbox which uses the default values. But to be sure ill leave it (I'll just reformulate the comment)